### DISCUSSION TITLE
Imgix with next/image, double compression or 1 less dsn lookup?

### DISCUSSION THREAD

**Discussion by robertschneiderman** [vercel/next.js#38928] [Help]
I noticed I have two ways I could use imgix with next/image:
1. Provide imgix URL's and let the images be hosted by next.js. I noticed copies of imgix photos will get created in `.next` build like so:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/16520011/180476276-237d5e48-467e-4eed-af2a-3e05d297fc71.png">

Pros: 1 less DNS lookup? (no resolving of https://domain.imgix.com)
Cons: Double optimization of images?

2. Set the loader to `imgix` in next.config.js, and use imgix URL's

Pros: 1 more DNS lookup?
Cons: No double optimization of images?

Note I put "?"'s after pros and cons because I'm not sure whether these things are true or not. Would next.js optimize an already optimized image and produce something low quality? Do I save a DNS lookup by using method 1 and is that reason enough for choosing it?

**Comment by wwwdepot** [comment]


**Reply by robertschneiderman** [reply]
hey @wwwdepot I think you may have wrote this in the wrong thread
